stable/jenkins/templates/NOTES.txt

```diff
@@ -18,7 +18,7 @@
   echo http://$SERVICE_IP:{{ .Values.Master.ServicePort }}/login
 
 {{- else if contains "ClusterIP"  .Values.Master.ServiceType }}
-  export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "component={{ template "jenkins.fullname" . }}-master" -o jsonpath="{.items[0].metadata.name}")
+  export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "component={{ .Release.Name }}-{{ .Values.Master.Name }}" -o jsonpath="{.items[0].metadata.name}")
```

Comment:
actually the pod seems to have label:
"{{ .Release.Name }}-{{ .Values.Master.Component }}"
(There is a difference between the deployment's label and it's containers)
https://github.com/helm/charts/blob/master/stable/jenkins/templates/jenkins-master-deployment.yaml

Reply:
True, I missed it.